vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -149,12 +149,20 @@ public function reconnect() {
 		if ( true === $connection_attempt ) {
 			if ( ! empty( $this->last_heartbeat['cache_site_id'] ) && (int) \Jetpack_Options::get_option( 'id' ) !== (int) $this->last_heartbeat['cache_site_id'] ) {
 				$this->send_alert( 'Alert: Jetpack was automatically reconnected, but the connection may have changed cache sites. Needs manual inspection.' );
-
 				return;
 			}
 
 			$this->send_alert( 'Jetpack was successfully (re)connected!' );
 
+			$akismet_connection_attempt = Connection_Pilot\Controls::connect_akismet();
+
+			if ( ! $akismet_connection_attempt ) {
+				$this->send_alert( 'Alert: Could not connect Akismet automatically.' );
+				return;
+			}
+
+			$this->send_alert( 'Akismet was successfuly connected!' );
```

Comment:
I think we can do without this being an alert. Just knowing when it fails seems sufficient.